Ingress/images/haproxy/pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: simple
+spec:
+  containers:
+  - image: gcr.io/google_containers/haproxy:0.0.1
```

Comment:
why 0.0.1? can we just make it 0.1 or 0.5 or were you following some other pattern (in which case please add a comment here and in the Makefile). Most of our gcr images are just a simple decimal.